new-integration-tests/src/test/java/oracle/weblogic/kubernetes/utils/K8sUtils.java

```diff
@@ -0,0 +1,114 @@
+// Copyright (c) 2020, Oracle Corporation and/or its affiliates.
+// Licensed under the Universal Permissive License v 1.0 as shown at https://oss.oracle.com/licenses/upl.
+
+package oracle.weblogic.kubernetes.utils;
+
+import java.util.HashMap;
+import java.util.Map;
+
+import io.kubernetes.client.custom.Quantity;
+import io.kubernetes.client.openapi.ApiException;
+import io.kubernetes.client.openapi.models.V1HostPathVolumeSource;
+import io.kubernetes.client.openapi.models.V1ObjectMeta;
+import io.kubernetes.client.openapi.models.V1ObjectMetaBuilder;
+import io.kubernetes.client.openapi.models.V1PersistentVolume;
+import io.kubernetes.client.openapi.models.V1PersistentVolumeClaim;
+import io.kubernetes.client.openapi.models.V1PersistentVolumeClaimSpec;
+import io.kubernetes.client.openapi.models.V1PersistentVolumeSpec;
+import io.kubernetes.client.openapi.models.V1ResourceRequirements;
+import oracle.weblogic.domain.PersistentVolume;
+import oracle.weblogic.domain.PersistentVolumeClaim;
+
+import static oracle.weblogic.kubernetes.extensions.LoggedTest.logger;
+
+/**
+ * Helper class to build Kubernetes objects.
+ */
+public class K8sUtils {
+
+  /**
+   * Creates a V1PersistentVolume object given a PersistentVolume POJO object.
+   *
+   * @param persistentVolume POJO object of PersistentVolume containing PV configuration data
+   * @return V1PersistentVolume object
+   * @throws ApiException when Kubernetes objects fails to be created
+   */
+  public static V1PersistentVolume createPVObject(PersistentVolume persistentVolume) throws ApiException {
```

Comment:
Remove this method

Reply:
The helper method I suggested was to create a V1PersistentVolumeClaim object from a V1PersistentVolume object, since these objects need similar fields.  This helper method would not be required though, as the tests could create both objects using similar constants.

Reply:
removed this class altogether